xl/version.py

```diff
@@ -41,7 +41,7 @@ def get_current_revision(directory):
     try:
         return subprocess.check_output([
             'git', 'rev-parse', '--short=7', 'HEAD'
-        ]).strip()
+        ], stderr=-1).strip()
```

Comment:
Python 3 has `subprocess.DEVNULL`, but it's not in 2.x. I think`file(os.devnull, 'w')` works.